### DISCUSSION TITLE
Make the colorpicker support different color models

### DISCUSSION THREAD

**Discussion by HitArrowLegend** [wintercms/winter#90] [Ideas]
I thought it would be nice to have the colorpicker allow HSL and RGB. HSL has the benefit of being easy to understand while I don't need to introduce RGB. HEX is fine but it doesn't allow easy adjustments (while HSL is kinda the current king in that domain).

**Comment by bennothommo** [comment]
@HitArrowLegend I'm fairly certain the color picker library in use does support those other formats, so I reckon it would be a trivial change to allow someone to define the format returned by the color picker. I've added an issue to have a look into implementing this at some point - https://github.com/wintercms/winter/issues/94.
